pandas/tests/io/json/test_deprecated_kwargs.py

```diff
@@ -0,0 +1,40 @@
+"""
+Tests for the deprecated keyword arguments for `read_json`.
+"""
+
+import pandas as pd
+import pandas.util.testing as tm
+from pandas.util.testing import assert_frame_equal
+
+from pandas.io.json import read_json
+
+
+def test_deprecated_kwargs():
+    """
```

Comment:
We don't typically use docstrings for tests so OK to remove here and below

Reply:
OK.